dev-manuals/architecture/overview.md

```diff
@@ -0,0 +1,28 @@
+# Overview
```

Comment:
Funny enough, that's why I created #165

Reply:
Oh, nice. I'll remove this file later and instead link to your page.